task/send-to-webhook-slack/0.1/send-to-webhook-slack.yaml

```diff
@@ -20,15 +21,32 @@ spec:
   - name: message
     type: string
     description: plain text message
+  - name: bot-name
+    type: string
+    description: plain text message
+    default: 'Tekton Bot'
```

Comment:
is there a way to make it conditional if the user doesn't want it?

Reply:
Yes there is. If you don't send a username the name of the bot becomes "Incoming webhook". See attached screenshot as an example. (The icon_emoji in our case was ":tekton:" which was an imported emoji. I think adding a username is better than providing none.

<img width="519" alt="Schermafbeelding 2021-06-02 om 19 31 58" src="https://user-images.githubusercontent.com/25788049/120526504-8d3f7d80-c3d9-11eb-8ccc-c334dd9c9332.png">


<img width="726" alt="Schermafbeelding 2021-06-02 om 19 32 02" src="https://user-images.githubusercontent.com/25788049/120526495-8c0e5080-c3d9-11eb-876d-56f55060d42a.png">